File: Web/Prakhar-08_PrakharRastogi_1923it1011_3/Introduction to Recursion/Print Decreasing.cpp
#include <iostream>
using namespace std;

void printDecreasing(int n){
    // write your code here
    if(n == 1) {
        cout << 1 << endl;
        return;
    }
    
    cout << n << endl;
    printDecreasing(n-1);
}

int main(){
    int n;
    cin >> n;
    printDecreasing(n);
}

File: AR-VR/arshivaastha_Astha_2024cse1096_2/IntroductionToRecursion/Print decreasing.java
import java.io.*;
import java.util.*;

public class Main {

    public static void main(String[] args) throws Exception {
        // write your code here
        Scanner in= new Scanner (System.in);
        int n= in.nextInt();
      printDecreasing(n);  
    }

    public static void printDecreasing(int n){
        if(n==0)
        return;
    System.out.println(n);
    printDecreasing(n-1);
    }

}